api/v1/helpers/nouns/subjects.js

```diff
@@ -159,6 +159,7 @@ module.exports = {
   fieldScopeMap: {
     samples: 'withSamples',
     hierarchy: 'hierarchy',
+    subjectHierarchy: 'subjectHierarchy',
   },
```

Comment:
The only suggestion I could come up with is "subjectOnlyHierarchy", but I leave that up to you.